installing/installing_openstack/installing-openstack-installer-ovnk8s.adoc

```diff
@@ -0,0 +1,46 @@
+[id="installing-openstack-installer-kuryr"]
+= Installing a cluster on OpenStack with OVNKubernetes
```

Comment:
```suggestion
= Installing a cluster on OpenStack with OVN-Kubernetes
```

Reply:
OVNKubernetes is the networkType, do you still want to rename it for the page title?

Reply:
That's my thought. "OVN-Kubernetes" seems preferred in the titles of other Red Hat docs.

Reply:
ack, done.